qiskit/circuit/library/__init__.py

```diff
@@ -559,6 +559,7 @@
     PauliTwoDesign,
     RealAmplitudes,
     EfficientSU2,
+    EvolvedOperatorAnsatz,
```

Comment:
Note that `EvolvedOperatorAnsatz` should be moved from line 333 to the `N-local` circuits section, so it will appear in the correct place in the API docs.

Reply:
Fixed.